{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings/base.py

```diff
@@ -92,6 +92,7 @@
 
 STATIC_ROOT = config('STATIC_ROOT', default=os.path.join(BASE_DIR, 'static'))
 STATIC_URL = config('STATIC_URL', '/static/')
+STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
```

Comment:
This is good as long as you're not also using django-pipeline. You have to combine the classes for that [like we did in bedrock](https://github.com/mozilla/bedrock/blob/532669e9ee15c20a65cd349a15cc998b46253844/bedrock/base/storage.py).